### PR TITLE
Update GetShellWithoutMonitorOnWindows.md

### PR DESCRIPTION
**Pull request by tjcannon**
Add instruction to create empty ssh file at root of boot volume, as required to enable headless ssh as of Rasbian Buster

### CODE REVIEW

File: doc/GetShellWithoutMonitorOnWindows.md [hunk 1]
```diff
@@ -26,6 +26,11 @@
     ```
 1. If you will use any public or insecure networks, be sure to update your password using the instructions at the end of this document.
 1. Enter the additional SSID and Pre-Shared Key of the WiFi network and repeat as necessary.
+1. Now let's make sure ssh is enabled for our first boot (mandatory for Rasbian Buster or newer):
```
Comment (marcone) [inline doc/GetShellWithoutMonitorOnWindows.md:29]: This was/is also needed for Stretch, and even for the last few Jessie releases that came before Stretch.

**Review by marcone — COMMENTED** [review]
Out of curiosity, how did you end up looking at this file? Did you not see the numerous recommendations to use the prebuilt image, or you just like building your own?

**Comment by tjcannon** [comment]
> Out of curiosity, how did you end up looking at this file? Did you not see the numerous recommendations to use the prebuilt image, or you just like building your own?

I ran a prebuilt image for a few months in my Model 3, then went back to "just a flash drive." Now that I'm back, I see the most recent build is >80 commits behind HEAD, so that made me want to give this a shot. And as long as I'm giving it a shot, I figured why not make things a little clearer for the next person to do the same?

And as for how I got to this file, I followed the link for getting initial shell (past all those warnings)

File: doc/GetShellWithoutMonitorOnWindows.md (hunk 1 above)
Comment (tjcannon) [reply doc/GetShellWithoutMonitorOnWindows.md:29]: Sounds good. I'll change the wording.

Comment (tjcannon) [reply doc/GetShellWithoutMonitorOnWindows.md:29]: Upon further investigation, the empty ssh file should have been added (but wasn't) by setup-piForHeadlessConfig.ps1, despite non-error log output. So I'm actually going to close this one too and look into potentially making the PS scripts more resilient. Sorry for all the unnecessary traffic, and thank you for the feedback. I'll think through things a bit more before submitting anything else.

**Pull request closed without merging**